UPGRADE-2.8.md

```diff
@@ -235,6 +235,23 @@ Form
    match the type returned by `getExtendedType` is now forbidden. Fix your
    implementation to define the right type.
 
+ * The alias option of the `form.type_extension` tag is deprecated in favor of
+   the `extended_type` option.
```

Comment:
Maybe also use `extended_type`/`extended-type` esp. since the following is code is xml which will cause confusion.